src/certifier.mak

```diff
@@ -41,18 +41,26 @@ INCLUDE=-I $(I) -I/usr/local/opt/openssl@1.1/include/ -I $(S)/sev-snp
 CFLAGS_COMMON = $(INCLUDE) -g -Wall -std=c++11 -Wno-unused-variable -D X64 -Wno-deprecated-declarations
 
 CFLAGS  = $(CFLAGS_COMMON) -O3
+CFLAGS_PIC =
 
 ifdef ENABLE_SEV
 CFLAGS  += -D SEV_SNP
 endif
 
+CFLAGS += $(CFLAGS_PIC)
+
 CC=g++
 LINK=g++
 
 # Point this to the right place, if you have to, based on your machine's install:
 # PROTO=/usr/local/bin/protoc
 PROTO=protoc
 AR=ar
+
+# Definitions needed for generating Python bindings
+SWIG=swig
+PY_INCLUDE = -I /usr/include/python3.10/
```

Comment:
This hard-coded path ... is generally not good. But I found that this seems to be the current-default rev-level.

It's working in CI, too.

Any suggestions on how-to create this -I definition using some symbols, to future-proof this area?

Reply:
Not an expert here. But this can't be good. My SGX machine for instance does not have 3.10. Let's do some digging.